endoreg_db/serializers/video_segmentation.py

```
@@ -400,7 +407,10 @@ def save(self):
         updated_segments = []
         new_entries = []
         existing_keys = set(existing_segments_dict.keys())
-        new_keys = set((float(seg["start_frame_number"]), float(seg["end_frame_number"])) for seg in new_segments)
+        new_keys = set(
```

Ruff (0.8.2)

Local variable `new_keys` is assigned to but never used

Remove assignment to unused variable `new_keys`

(F841)